index.js

```diff
@@ -63,8 +63,11 @@ module.exports = {
     return path.join(__dirname, 'blueprints');
   },
 
-  included: function(app) {
-    this._super.included(app);
+  included: function included(app, parentAddon) {
+    var target = (parentAddon || app);
+    this._super.included(target);
```

Comment:
Sorry for the annoyance, can you tweak this to `this._super.included.call(this, target)`?